app-engine/src/main/java/be/cytomine/appengine/services/AppStoreService.java

```diff
@@ -0,0 +1,86 @@
+package be.cytomine.appengine.services;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import be.cytomine.appengine.dto.responses.errors.AppEngineError;
+import be.cytomine.appengine.dto.responses.errors.ErrorBuilder;
+import be.cytomine.appengine.dto.responses.errors.ErrorCode;
+import be.cytomine.appengine.exceptions.AppStoreNotFoundException;
+import be.cytomine.appengine.exceptions.ValidationException;
+import be.cytomine.appengine.models.store.AppStore;
+import be.cytomine.appengine.repositories.AppStoreRepository;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class AppStoreService {
+
+    private final AppStoreRepository appStoreRepository;
+
+    public Optional<AppStore> findDefault() {
+        return appStoreRepository.findByDefaultStoreIsTrue();
+    }
+
+    public List<AppStore> list() {
+        return appStoreRepository.findAll();
+    }
+
+    public void delete(UUID id) {
+        appStoreRepository.deleteById(id);
+    }
+
+    @Transactional
+    public void makeDefault(UUID id) throws AppStoreNotFoundException {
+        log.info("make default {}", id);
```

Comment:
I suggest making the logging more useful and precise something like


```suggestion
        log.info("Setting store [{}] as default", id);
```

Reply:
okay